### PR TITLE
limit nagios message lenght

### PR DESCRIPTION
**Pull request by JensTimmerman**
this is probably not needed at all and can be closed

### CODE REVIEW

File: lib/vsc/utils/nagios.py [hunk 1]
```diff
@@ -60,20 +60,27 @@
...
+    @param metrics: Metrics for nagios, used to create graphs
     """
     (exit_code, text) = code
-    print "%s %s" % (text, message)
-    log.info("Nagios report %s: %s" % (text, message))
+    message = message.split('|')
+    if len(message) > 1:
+        message = message[0]
+        metrics = ' | %s' % message[1]
+    print "%s %s%s" % (text, message[:NAGIOS_MAX_MESSAGE_LENGTH], metrics)
```
Comment (stdweird) [inline lib/vsc/utils/nagios.py]: it's better to replace all cutoff characters with a clear value like `CUTOFF` (and when there's a cutoff, report it)

**Comment by wdpypere** [comment]
@JensTimmerman please update?

**Comment by JensTimmerman** [comment]
not needed I think

**Comment by wdpypere** [comment]
still needed.

**Comment by wdpypere** [comment]
@JensTimmerman version bump?

File: lib/vsc/utils/nagios.py [hunk 2]
```diff
@@ -60,20 +60,31 @@
...
+    message = message.split('|')
+    msg = message[0]
+    if len(message) > 1:
+        metrics = '|%s' % message[1]
+    if len(msg) > NAGIOS_MAX_MESSAGE_LENGTH:
+        # log long message but print truncated message
+        log.info("Nagios report %s: %s%s" % (text, msg, metrics))
+        msg = msg[:NAGIOS_MAX_MESSAGE_LENGTH-3] + '...'
+
+    print "%s %s%s" % (text, msg, metrics)
```
Comment (wdpypere) [inline lib/vsc/utils/nagios.py]: I don't like the variable name 'text' for a the textual representation of a nagios exit code. It confused me.

**Pull request merged**